### PR TITLE
Backend/API: Implementation of ADD and MUL operations in the compile()

### PR DESCRIPTION
**Pull request by shssf**
This is proposal for first ADD and MUL operations implementation.
The class IntelGPUBackend::FunctionInstance is implemented in PR #1199 
Sources are formatted with `${build}/llvm/bin/clang-format -style=file *.cpp`

@rkimballn1 Please review.

### CODE REVIEW

**Comment by shssf** [comment]
Could someone please recommend me pre-checking procedure to avoid such things? I did automatic source formatting described above.

```
1 of 759 source files failed the code-format check:
    src/ngraph/runtime/intelgpu/intelgpu_layout.hpp
test/CMakeFiles/style-check.dir/build.make:57: recipe for target 'test/CMakeFiles/style-check' failed
make[3]: *** [test/CMakeFiles/style-check] Error 1
```

**Comment by diyessi** [comment]
I  run “make check” but occasionally the style checker behaves differently on the CI platform and my laptop.

Scott Cyphers

**Comment by rkimballn1** [comment]
The easiest way to avoid the dreaded style error is to run main/apply_code_format.sh on a machine with the correct clang-format installed. Once you get your environment setup it gets to be less of a problem.

File: src/ngraph/runtime/intelgpu/intelgpu_backend.cpp [hunk 1]
```diff
@@ -14,12 +14,43 @@
...
 using namespace ngraph;
 
+void do_eltwise_operation(cldnn::topology& topology,
+                          const shared_ptr<Node>& op,
+                          cldnn::eltwise_mode mode)
+{
+    if (op->get_output_size() != 1 || op->get_input_size() != 2)
+    {
+        ostringstream os;
+        os << "eltwise operation \"" << op->description() << "\" input and output sizes mismatch.";
```
Comment (rkimballn1) [inline src/ngraph/runtime/intelgpu/intelgpu_backend.cpp]: It is nice to know what the sizes where.
Spell out the word `Elementwise` as this message is intended for someone who is not you.

**Comment by shssf** [comment]
@rkimballn1 I moved parameters check to standalone function to avoid code duplication.
Also, I added "Result" operation handling.

**Pull request merged**